src/main/java/org/spongepowered/api/item/inventory/crafting/CraftingGridInventory.java

```diff
@@ -0,0 +1,21 @@
+package org.spongepowered.api.item.inventory.crafting;
+
+import org.spongepowered.api.item.inventory.type.GridInventory;
+import org.spongepowered.api.item.recipe.crafting.CraftingRecipe;
+import org.spongepowered.api.world.World;
+
+import java.util.Optional;
+
+/**
+ * A CraftingGridInventory represents the inventory of something that can craft
+ * items. This is excluding the Result slot.
+ */
+public interface CraftingGridInventory extends GridInventory {
```

Comment:
This is the equivalent to InventoryCrafting.

CraftingInventory is a view on InventoryCrafting and the ResultSlot